docs/localization/vtex-io-documentation-paymentprovider-builder.md

```diff
@@ -0,0 +1,34 @@
+---
+title: PaymentProvider builder"
```

Comment:
I believe there's a quote missing at the beginning

Reply:
Yes, it is missing. Thanks for noticing!

```suggestion
title: "PaymentProvider builder"
```